curp/tests/common/curp_group.rs

```diff
@@ -40,7 +40,6 @@ pub struct CurpNode {
     pub exe_rx: mpsc::UnboundedReceiver<(TestCommand, TestCommandResult)>,
     pub as_rx: mpsc::UnboundedReceiver<(TestCommand, LogIndex)>,
     pub store: Arc<Engine>,
-    pub rt: Runtime,
```

Comment:
I think a seperate runtime is needed to shutdown the RPC server when dropped. Otherwise the server may still writing to the files after `stop` is called.